src/library/dlgtrackinfo.cpp

```diff
@@ -545,6 +566,14 @@ void DlgTrackInfo::slotBpmConstChanged(int state) {
     }
 }
 
+void DlgTrackInfo::slotBpmLockChanged(int state) {
+    if (state == Qt::Checked) {
```

Comment:
enableBpmControls(state == Qt::Checked);

Since this slot is not needed elsewhere I suggest, replacing this single-line function with a lambda in connect() to reduce the noise and resolve the unclear naming issues.